examples/api/test/material/bottom_app_bar/bottom_app_bar.1_test.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/bottom_app_bar/bottom_app_bar.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets(
+      'BottomAppBarDemo shows FloatingActionButton and responds to toggle',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(const example.BottomAppBarDemo());
+
+    expect(find.byType(FloatingActionButton), findsOneWidget);
+
+    // Tap the 'Floating Action Button' switch to hide the FAB.
+    await tester.tap(find.byType(SwitchListTile).first);
+    await tester.pumpAndSettle();
+
+    expect(find.byType(FloatingActionButton), findsNothing);
+  });
```

Comment:
```suggestion
  testWidgets(
    'BottomAppBarDemo shows FloatingActionButton and responds to toggle',
    (WidgetTester tester) async {
      await tester.pumpWidget(const example.BottomAppBarDemo());

      expect(find.byType(FloatingActionButton), findsOneWidget);

      // Tap the 'Floating Action Button' switch to hide the FAB.
      await tester.tap(find.byType(SwitchListTile).first);
      await tester.pumpAndSettle();

      expect(find.byType(FloatingActionButton), findsNothing);
    },
  );
```

Reply:
For tests with a long description prefer the format I suggested.
Thank you for applying this format to all the other tests with a long description (you can also use it for all tests if you want).